test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -5234,6 +5234,17 @@ def forward(self, x):
         self.run_test(TensorFactory(), x, test_with_inputs=[another_x],
                       input_names=["input_1"], dynamic_axes={"input_1": [0, 1, 2]})
 
+    @skipIfUnsupportedMinOpsetVersion(13)
+    def test_diagonal(self):
+        class TensorFactory(torch.nn.Module):
+            def forward(self, x):
+                return torch.diagonal()
```

Comment:
The documentation says torch.diagonal requires an argument:
https://pytorch.org/docs/master/generated/torch.diagonal.html

Did you forget to pass in x as the first arg?

Does this test pass locally for you?

Reply:
Added x as an argument